### ISSUE TITLE
Inject `.profile` data into the execution environment rather than sourcing the file

### ISSUE THREAD

**Issue by davidcassany** [OSInside/kiwi#1538]
## Problem description

`.profile` file is being written and created to include environment variables into certain scripts like `config.sh` where it is being sourced. However python provides us all the tools to actually make sure this environment variables are loaded into the environment being that is being called without having to source anything within the script. Basically we could get rid of the `.profile` file.

## Expected behaviour

No need to source `.profile` file in `config.sh` to get the KIWI environment variables.

Related to #1475

**Comment by schaefi** [comment]
I'm working on this one

**Comment by schaefi** [comment]
We have already done this. Scripts are called with the following code

```python
caller_environment.update(profile.get_settings())
config_script = Command.call(command, caller_environment)
process = CommandProcess(
    command=config_script, log_topic='Calling ' + name + ' script'
)
```

I have done some tests to verify that this is really working and it does work well. I wrote a simple script **config.sh**

```
env > foo
```

and foo contained all kiwi profile variables

I have tested scripts with 'x' bit set and scripts without. Scripts without execution bit are called through bash. In both cases the caller environment has all the kiwi profile variables
